pinot-segment-local/src/main/java/org/apache/pinot/segment/local/aggregator/DistinctCountHLLPlusValueAggregator.java

```diff
@@ -31,6 +31,7 @@
 
 public class DistinctCountHLLPlusValueAggregator implements ValueAggregator<Object, HyperLogLogPlus> {
   public static final DataType AGGREGATED_VALUE_TYPE = DataType.BYTES;
+  public static final AggregationFunctionType AGGREGATION_FUNCTION_TYPE = AggregationFunctionType.DISTINCTCOUNTHLL;
```

Comment:
This is incorrect. It should be `DISTINCTCOUNTHLLPLUS`, which explains why the test failed

Reply:
I see. The old one is a bug, and we can just fix it here. There should be no consequence

Reply:
Fixed in https://github.com/apache/pinot/pull/12164/commits/722d89b3f88a7fe3ec0d97c94f88add2330f980e.